src/handlebars/index.handlebars

```diff
@@ -51,7 +51,7 @@
 
     <a href="./releases.html" class="dl-button dl-thin-button a-button fadeIn invisible" id="dl-other">
       <div>
-        <span>Other platforms</span>
+        <span>Other downloads</span>
```

Comment:
Not sure about this change - appreciate that you're trying to tell people it's other platforms and/or other ways of installing. But I felt that _other platforms_ is stronger and at this stage, in order to get to a `tar.gz` I now have to double bounce through the releases.

I wonder if we show both the installer and the binary downloads here...

Reply:
reverting to platforms for now then.